src/test/metrics_flow_test.cpp

```diff
@@ -598,6 +785,10 @@ std::string MetricFlowTest::prepareConfigContent() {
            R"(",")" + METRIC_NAME_STREAMS +
            R"(",")" + METRIC_NAME_INFERENCE_TIME +
            R"(",")" + METRIC_NAME_WAIT_FOR_INFER_REQ_TIME +
+           R"(",")" + METRIC_NAME_CURRENT_GRAPHS +
```

Comment:
Is there a functional test for that metric.

Reply:
Added